scripts/install/deploy/install_db.sh

```diff
@@ -27,3 +21,6 @@ fi
  else
    echo "Postgres already running"
  fi
+
+sudo cp timeless_pg.service /lib/systemd/system/
+sudo systemctl start timeless_pg.service
```

Comment:
@marceloamadeu I think it should be only after installation and before start.
And I think 
`service postgresql start` and `sudo systemctl start timeless_pg.service` do the same, don't they?

Reply:
@vladarefiev It is true. Do you think we should include in `rultor.yml` then?

Reply:
@marceloamadeu let's ask @emilianodellacasa

Reply:
@marceloamadeu @vladarefiev This script is already called from rultor, actually we already start postres we only need to add the service to boot

Reply:
@vladarefiev @emilianodellacasa The form this is now is the same as issue #300. But in this case for `postgres`. Are you going to change anything here? What is missing?
tks

Reply:
@marceloamadeu @vladarefiev In case of issue #300, the command is 'sudo systemctl enable' to tell redis to start at boot. In this case you are giving 'sudo systemctl start', simply starting the service without enabling autostart. This is the difference